crashlytics/src/AndroidImpl.cs

```diff
@@ -141,6 +142,39 @@ internal class AndroidImpl : Impl {
         }, "LogException");
     }
 
+    public override void LogExceptionAsFatal(Exception exception) {
+      var loggedException = LoggedException.FromException(exception);
+      Dictionary<string, string>[] parsedStackTrace = loggedException.ParsedStackTrace;
+      
+      if (parsedStackTrace.Length == 0) {
+        // if for some reason we don't get stack trace from exception, we add current stack trace in
+        var currentStackTrace = System.Environment.StackTrace;
+        LoggedException loggedExceptionWithCurrentStackTrace = new LoggedException(loggedException.Name, loggedException.Message, currentStackTrace);
+        parsedStackTrace = loggedExceptionWithCurrentStackTrace.ParsedStackTrace;
+
+        if (parsedStackTrace.Length > 2) {
+          // remove RecordCustomException and System.Environment.StackTrace frame for fault blame on crashlytics sdk
```

Comment:
@mrober I directly copy this block from iOSUmpl.cs and then realized Android does not have `RecordCustomException` this function. Can you confirm if removing 1 stack frame is good enough for android?

Reply:
I tuned it a bit in 92a567, before: https://screenshot.googleplex.com/4JnexnXpYZ6dbzE.png after: https://screenshot.googleplex.com/4qc8Fy48ZUfQKpG.png